lib/src/badge_positioned.dart

```diff
@@ -30,10 +31,10 @@ class BadgePositioned extends StatelessWidget {
       );
     }
     return PositionedDirectional(
-      top: position.top,
-      end: position.end,
-      bottom: position.bottom,
-      start: position.start,
+      top: position!.top,
+      end: position!.end,
+      bottom: position!.bottom,
+      start: position!.start,
```

Comment:
I think these aren't safe. Would not `position?.top` be better?

Reply:
These were safe because if `position` is `null` something else is returned.
By introducing a local variable the `!` is not needed anymore.

Reply:
yes, you're right! I overlooked the null check 👍 but with the local variable, it looks cleaner